### PR TITLE
Adding French translations for dental-insurance ITA page

### PR DESCRIPTION
**Pull request by nibivi77**
### Related Azure Boards Work Items
[AB#4757](https://dev.azure.com/DTS-STN/1fc40a8f-28cf-47bc-b6e4-1c234bd06177/_workitems/edit/4757)

### CODE REVIEW

File: frontend/public/locales/fr/renew-ita.json [hunk 1]
```diff
@@ -153,40 +153,40 @@
     }
   },
   "dental-insurance": {
-    "title": "Access to other dental insurance",
-    "legend": "Do you have access to dental insurance or coverage:",
+    "title": "Accès à d'autres assurances dentaires",
+    "legend": "Avoir accès à une assurance dentaire signifie que vous avez accès à une assurance ou à une couverture dentaire :",
```
Comment (gregory-j-baker) [inline frontend/public/locales/fr/renew-ita.json:157]:
```suggestion
    "legend": "Avoir accès à une assurance dentaire signifie que vous avez accès à une assurance ou à une couverture dentaire\u00a0:",
```

**Review by gregory-j-baker — DISMISSED** [review]
Approved with one comment.

File: frontend/public/locales/fr/renew-ita.json [hunk 2]
```diff
@@ -153,40 +153,40 @@
...
+      "continue": "Continuer",
+      "cancel-btn": "Annuler",
+      "save-btn": "sauvegarder"
     },
     "detail": {
       "additional-info": {
-        "title": "Additional information",
-        "eligible": "Exception: You may still be eligible if you are retired and:",
+        "title": "Renseignements supplémentaires",
+        "eligible": "Exception: Vous pourriez toujours être admissible si vous avez pris votre retraite et:",
```
Comment (nicholas-ly) [inline frontend/public/locales/fr/renew-ita.json:175]:
```suggestion
        "eligible": "Exception: Vous pourriez toujours être admissible si vous avez pris votre retraite et\u00a0:",
```

File: frontend/public/locales/fr/renew-ita.json [hunk 3]
```diff
@@ -153,40 +153,40 @@
...
+          "opted": "vous avez renoncé aux prestations de retraite avant le 11 décembre 2023, et",
+          "cannot-opt": "vous ne pouvez pas vous réinscrire en vertu des règles relatives aux régimes de retraite"
         },
-        "not-eligible": "You are still considered to have access to dental insurance or coverage if you choose to opt out of benefits provided through your or a family member's:",
-        "not-eligible-employer": "employer",
-        "not-eligible-pension": "pension (see exception)",
-        "not-eligible-organization": "professional or student organization.",
-        "not-eligible-note": "As such, you are not eligible for the Canadian Dental Care Plan.",
-        "not-eligible-purchased": "If you purchased your current dental insurance policy privately on your own, you are not eligible for the Canadian Dental Care Plan while that coverage is in effect."
+        "not-eligible": "Vous êtes toujours considéré comme ayant accès à l'assurance ou à la couverture dentaire si vous choisissez de ne pas bénéficier des prestations fournies par votre assurance ou celle d'un membre de votre famille:",
```
Comment (nicholas-ly) [inline frontend/public/locales/fr/renew-ita.json:180]:
```suggestion
        "not-eligible": "Vous êtes toujours considéré comme ayant accès à l'assurance ou à la couverture dentaire si vous choisissez de ne pas bénéficier des prestations fournies par votre assurance ou celle d'un membre de votre famille\u00a0:",
```

**Review by nicholas-ly — APPROVED** [review]

**Pull request merged**